### ISSUE TITLE
Add Community Files ( CONTRIBUTIONS.md , CODE_OF_CONDUCT.md, etc ) and Define a Suitable File Organization Structure

### ISSUE THREAD

**Issue by Ansh-Sarkar** [OmarBazaraa/Competitive-Programming#5]
- Add Community Files and Docs such as ```CONTRIBUTIONS.md``` , ```CODE_OF_CONDUCT.md```, etc for inviting contributions.
- In the documentation, decide upon a folder / directory structure in order to maintain consistency across the project.

**Comment by Ansh-Sarkar** [comment]
I have made the above mentioned contributions. Shall I proceed with opening a pull request for this issue ? Thought it would be a good thing to open an issue first and then link it to the PR for better documentation and logging purpose.

**Comment by Ansh-Sarkar** [comment]
Also, should I add an MIT license to the project ? Do let me know.

**Comment by OmarBazaraa** [comment]
Yes, please create a PR for the changes. And thanks for your contributions.

**Comment by Ansh-Sarkar** [comment]
Sure. Creating a PR now. Will continue contributing as much as possible. Thanks !

**Comment by Ansh-Sarkar** [comment]
Have created the PR. Will be updating the rest of the documents one by one and keep contributing. Do let me know me know if we should be adding a LICENSE.md file to this repo.
